CyberHealth/users/views.py

```diff
@@ -7,7 +7,8 @@
 import uuid
 
 
-def send_user_notification(user_details, user_token, template_id='63d94931-3b5a-42dc-ba0d-06b40902298b'):
+def send_user_notification(user_details, user_token,
+                           template_id='63d94931-3b5a-42dc-ba0d-06b40902298b'):
```

Comment:
What is this ID? Is this something that should be moved to environment variable or something...?

Reply:
I'm not sure, it was on develop. @joel-stobart do you have any thoughts on this one?

Reply:
It's the Gov.uk Notify notification id - it's which template we should use (i.e. Forgot password email or activation email )

Reply:
It shouldn't be environmental

Reply:
It could be wrappered up in the future. But good enough for now.